src/services/user.service.js

```diff
@@ -23,7 +23,7 @@ import Alert from '@/model/Alert.model'
 export const UserService = {
 
   getUserProfile () {
-    return axios.get(`${window.location.pathname}/userprofile`).then((response) => {
+    return axios.get(`${window.location.pathname}userprofile`).then((response) => {
```

Comment:
I think this change is not necessary any longer too (it was related to the `NODE_ENV=offline` that we had to add I think?).

I think the `pathname` may contain a file name, not a `/` (slash), in which case we need to add the slash (I think).

- https://www.w3schools.com/js/js_window_location.asp (search pathname)
- https://developer.mozilla.org/en-US/docs/Web/API/Location